src/CMakeLists.txt

```diff
@@ -19,8 +19,8 @@
 #
 # src/CMakeLists.txt
 #
-set (SUPPL_NAME core)
-set (SUPPL_PURPOSE "GMT core: The main modules of the Generic Mapping Tools")
+set (SHARED_LIB_NAME core)
+set (SHARED_LIB_PURPOSE "GMT core: The main modules of the Generic Mapping Tools")
```

Comment:
Could you move these two lines to Line 340?

Reply:
OK